src/ng/filter/orderBy.js

```diff
@@ -690,7 +695,7 @@ function orderByFilter($parse) {
         result = value1 < value2 ? -1 : 1;
       }
     } else {
-      result = type1 < type2 ? -1 : 1;
+      result = (type1 === 'undefined') ? 1 : (type2 === 'undefined') ? -1 : (type1 === 'null') ? 1 : (type2 === 'null') ? -1 : (type1 < type2) ? -1 : 1;
```

Comment:
This line is still >100 chars long 🤔

Reply:
What about using the following instead of wrapping it at 100:

```
result = (type1 === 'undefined') ? 1 :
        (type2 === 'undefined') ? -1 :
        (type1 === 'null') ? 1 :
        (type2 === 'null') ? -1 :
        (type1 < type2) ? -1 : 1;
```